errors.go

```diff
@@ -22,6 +22,9 @@ import (
 
 // common error objects used for argument checking
 var (
+	ErrNeedPod         = errors.New("Pod must be specified")
 	ErrNeedPodID       = errors.New("Pod ID cannot be empty")
 	ErrNeedContainerID = errors.New("Container ID cannot be empty")
+	ErrNeedFile        = errors.New("file cannot be empty")
```

Comment:
Hi @wcwxyz - thanks for reminding me about this. I had planned for these errors to refer to the name of the object originally. `file` didn't fit because that error is used for a file path. But looking at this list again, it's all rather inconsistent since it should be `containerID` and `podID`. I think as you say, just capitalising `file` would be the simplest fix ;)